tests/cli-misc.rs

```diff
@@ -866,8 +866,8 @@ fn add_remove_component() {
 #[test]
 fn which() {
     setup(&|config| {
-        let path_1 = config.customdir.join("custom-1");
```

Comment:
I wonder why are these changes necessary?

Reply:
Certainly this additional leading underscore isn't needed.  However the conversion is needed afaict.